lib/daru/view/adapters/datatables.rb

```diff
@@ -1,6 +1,7 @@
 require 'daru'
 require 'securerandom'
 require 'erb'
+require 'daru/data_tables'
```

Comment:
Need to check how tesstcases has written related to datatables (or may be not written). When I was running IRuby notebook and tried the adapter datatables it was not working.

After requiring the `daru/data_tables` it worked fine. Need to write testcase for it in separate PR.